frontend/src/container/PipelinePage/AddNewPipline.tsx

```diff
@@ -0,0 +1,31 @@
+import { PlusOutlined } from '@ant-design/icons';
+import TextToolTip from 'components/TextToolTip';
+import React, { useState } from 'react';
+
+import NewPipline from './NewPipeline';
+import { Button, ButtonContainer } from './styles';
+
+function AddNewPipline(): JSX.Element {
+	const [addPipeline, setNewAddPiplines] = useState<boolean>(false);
+
+	return (
+		<ButtonContainer>
+			<TextToolTip text="add Piplines" />
+			{addPipeline && (
+				<NewPipline
+					addPipeline={addPipeline}
+					setNewAddPiplines={setNewAddPiplines}
+				/>
+			)}
+			<Button
+				icon={<PlusOutlined />}
+				onClick={(): void => setNewAddPiplines(true)}
```

Comment:
inline function

Reply:
@palashgdev - You want me to create separate function?